generators/app/index.js

```diff
@@ -32,7 +32,7 @@ module.exports = (props) => {
   } else {
     const repo = `https://github.com/${props.templateRepo}.git`
     // clone the template repo in the current directory
-    exec('git', ['clone', repo], {}, function () {
+    exec('git', ['clone', '-q', repo], {}, function () {
```

Comment:
when tests were not passing, it was hard to debug them with all the git output there, so I make it `quiet`.